qa/suites/fs/one-role/tasks/mgr-osd-full.yaml

```diff
@@ -0,0 +1,21 @@
+overrides:
+  ceph:
+    conf:
+      global:
+        osd_pool_default_size: 1
+        osd_pool_default_min_size: 1
+      client:
+        debug ms: 1
+        debug client: 20
+        client check pool perm: false
+      mds:
+        debug ms: 1
+        debug mds: 20
+      osd: # force bluestore since it's required for ec overwrites
+        osd objectstore: bluestore
+        bluestore block size: 1073741824
+tasks:
+- workunit:
+    clients:
+      client.0:
+        - fs/misc/osd_full_mgr_hang.sh
```

Comment:
let's rename this to something more speicifc: `fs/full/subvolume_rm.sh`.

We'll probably add other tests someday.

Reply:
Done